frontend/src/generate-static-html.js

```diff
@@ -1,24 +1,42 @@
 
```

Comment:
You're right, it shouldn't be shipped to the web. I wasn't really sure where to put it! I'll figure out where to put this and make sure it doesn't get shipped before this lands.